File: services/api-principal/src/main/java/com/guminteligencia/ura_chatbot_ia/infrastructure/repository/entity/VendedorEntity.java
package com.guminteligencia.ura_chatbot_ia.infrastructure.repository.entity;

import com.guminteligencia.ura_chatbot_ia.domain.Prioridade;
import jakarta.persistence.*;
import lombok.*;

@Entity(name = "Vendedor")
@Table(name = "vendedores")
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Builder
@ToString
public class VendedorEntity {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id_vendedor")
    private Long id;
    private String nome;
    private String telefone;
    private Boolean inativo;

    @Column(name = "id_vendedor_crm")
    private Integer idVendedorCrm;

    @Embedded
    private Prioridade prioridade;

    private Boolean padrao;
}

File: services/api-agente/src/infrastructure/entity/vendedor_entity.py
from sqlalchemy import Column, BigInteger, String, Boolean
from src.config.base_class import Base

class VendedorEntity(Base):
    __tablename__ = "vendedores"

    id_vendedor = Column(BigInteger, primary_key=True, autoincrement=True)
    nome = Column(String(255), nullable=True)
    telefone = Column(String(255), nullable=True)
    inativo = Column(Boolean, nullable=True)